rundevtests.sh

```diff
@@ -84,4 +85,6 @@ if [ "$SETUP_TEST_USERS" = "true" ]; then
 fi
 sleep 10
 
-k6 run --no-usage-report --out dashboard - < <(cat ./test/${K6_TEST_FILE}.js)
\ No newline at end of file
+export K6_PROMETHEUS_RW_HTTP_HEADERS="X-Scope-OrgID:FASE,X-Prometheus-Remote-Write-Version:0.1.0,Authorization:Basic $MIMIR_CREDENTIAL"
+export K6_PROMETHEUS_RW_TREND_STATS="p(95),p(99),min,max,sum,avg,med"
+k6 run --tag testid=${K6_TEST_ID} --no-usage-report --out experimental-prometheus-rw ./test/${K6_TEST_FILE}.js
```

Comment:
We may need to make this optional for local test runs. Maybe add an update in the read me file so someone can set it up when they need to do that.